TranformingData/transformingDataExample.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3.6
+
+import dask.dataframe as dd
+import pandas as pd
+import sys
+import pickle
+import matplotlib.pyplot as plt
+
+def dataFrameNames():
+    return [
+        "x1",     # 1 gamma detected x position [cm]
+        "y1",     # 1 gamma detected y position [cm]
+        "z1",     # 1 gamma detected z position [cm]
+        "t1",     # 1 gamma detection time [ps]
+        "x2",     # 2 gamma detected x position [cm]
+        "y2",     # 2 gamma detected y position [cm]
+        "z2",     # 2 gamma detected z position [cm]
+        "t2",     # 2 gamma detection time [ps]
+        "vol1",   # 1 gamma volume ID
+        "vol2",   # 2 gamma volume ID
+        "e1",     # 1 gamma energy loss during detection [keV]
+        "e2",     # 2 gamma energy loss during detection [keV]
+        "class",  # Type of coincidence(1-true, 2-phantom-scattered, 3-detector-scattered, 4-accidental)
+        "sX1",    # 1 gamma emission x position [cm]
+        "sY1",    # 1 gamma emission y position [cm]
+        "sZ1",     # 1 gamma emission z position [cm]
+        "dt",     # Detection times difference
+        "rX1",    # Reconstruction point - X cord
+        "rY1",    # Reconstruction point - Y cord
+        "rZ1",    # Reconstruction point - Z cord
+        "rError", # Difference beetwen source point and recontructed point
+        "volD",   # Volumes indexes difference
+        "lorL",   # LOR length
+        "deg3D",  # Angle beetwen lines (in XYZ geometry) connecting detection points with the center of detector
+        "deg2D",  # Angle beetwen lines (in XY geometry) connecting detection points with the center of detector
+        "rL",     # Distance beetween reconstructed point and the center of detector
+        "eSum"    # Sum of the detecions energies
+    ]
+
+pathToFile = '/mnt/home/jbielecki1/NEMA/NEMA_IQ_384str_N0_1000_COINCIDENCES_PREPARED_part00'
```

Comment:
Better default would be working directory ...

Reply:
https://github.com/JPETTomography/j-pet-multiphoton-classification/pull/2/commits/6ae9978ab7b2844a0dedb0df19e71f88f03c0797